### PR TITLE
Setting up nightly postgres database backup

### PR DESCRIPTION
**Pull request by manishkumr**
This PR adds a service to create a nightly postgres database backup for XNAT

### CODE REVIEW

File: configure.sh [hunk 1]
```diff
@@ -28,6 +28,12 @@ else
     echo "Loaded saved value for CINDER_DIR=$CINDER_DIR"
 fi
 
+if [ -z "$BACKUP_DIR" ]; then
+    read -p 'Please enter postgres backup directory (BACKUP_DIR) :' CINDER_DIR
```
Comment (tclose) [inline configure.sh]:
```suggestion
    read -p 'Please enter postgres backup directory (BACKUP_DIR) :' BACKUP_DIR
```

File: docker-compose.yml [hunk 2]
```diff
@@ -9,17 +9,8 @@ services:
        - "8081:8080"
        - "8104:8104"
      volumes:
-       - $CINDER_DIR/webapps:/opt/tomcat/webapps
```
Comment (tclose) [inline docker-compose.yml:12]: What was your thinking re moving these volumes into the override DC?

**Review by tclose — COMMENTED** [review]
Thanks a lot Manish, that all looks really good.

The only thing I am not sure of is moving the CINDER_DIR DATA_DIR volumes into the docker-compose.override.yml. I was thinking it would be good to keep the override docker-compose.override.yml as SSL, so if you don't have certs you could just use the base docker-compose.yml. What was your thinking on this?

**Comment by manishkumr** [comment]
> Thanks a lot Manish, that all looks really good.
> 
> The only thing I am not sure of is moving the CINDER_DIR DATA_DIR volumes into the docker-compose.override.yml. I was thinking it would be good to keep the override docker-compose.override.yml as SSL, so if you don't have certs you could just use the base docker-compose.yml. What was your thinking on this?

Yes. My idea is that the settings in the base docker-compose file should be sufficient enough to bring a XNAT instance by running `docker-compose -f docker-compose.yml up -d`
docker-compose.override.yml will be used for extending/modifying any base configuration like installing ssl certs, volume mounts, backup schedules etc. These override would be picked by docker-compose while running  `docker-compose up -d`

File: docker-compose.yml [hunk 3]
```diff
@@ -41,6 +32,16 @@ services:
        - $CINDER_DIR/postgres:/var/lib/postgresql/data
```
Comment (tclose) [inline docker-compose.yml]: Should this be in the overrides YAML as well?

**Comment by tclose** [comment]
Okay, that sounds good.

One last thing regarding the "cinder" terminology, I did a few searches and couldn't find it used anywhere. Is there a more intuitive name we could use instead that has a wider usage?

File: docker-compose.yml (hunk 3 above)
Comment (manishkumr) [reply docker-compose.yml]: Yes. I will update the PR.

**Comment by manishkumr** [comment]
> Okay, that sounds good.
> 
> One last thing regarding the "cinder" terminology, I did a few searches and couldn't find it used anywhere. Is there a more intuitive name we could use instead that has a wider usage?

Cinder is just a service for openstack to present block storage.  Not sure what else we could use, but may be something representing the type of data it is going to hold, similar to DATA_DIR and BACKUP_DIR .

**Pull request merged**